### PR TITLE
Import/Export UI

### PR DESCRIPTION
**Pull request by Alotor**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: frontend/resources/styles/main/partials/modal.scss [hunk 1]
```diff
@@ -202,12 +202,338 @@
...
+  }
+
+  .option-container {
+    display: block;
+    position: relative;
+    padding-left: 40px;
+    padding-right: 1rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    // margin-bottom: 12px;
```
Comment (hirunatan) [inline frontend/resources/styles/main/partials/modal.scss]: remove this?

Comment (Alotor) [reply frontend/resources/styles/main/partials/modal.scss]: ok

File: frontend/src/app/main/ui/dashboard/import.cljs [hunk 2]
```diff
@@ -49,6 +50,264 @@
...
+(defn set-analyze-error
+  [files uri]
+  (->> files
+       (mapv (fn [file]
+               (cond-> file
+                 (= uri (:uri file))
+                 (assoc :status :analyze-error))))))
+
+(defn set-analyze-result [files uri data]
+  (let [exiting-files? (into #{} (->> files (map :file-id) (filter some?)))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/import.cljs]: exiSting?

Comment (hirunatan) [reply frontend/src/app/main/ui/dashboard/import.cljs]: or exiTTing? :stuck_out_tongue:

Comment (Alotor) [reply frontend/src/app/main/ui/dashboard/import.cljs]: fixed

File: frontend/translations/en.po [hunk 3]
```diff
@@ -2696,4 +2699,43 @@ msgid "workspace.updates.update"
 msgstr "Update"
 
 msgid "workspace.viewport.click-to-close-path"
-msgstr "Click to close the path"
\ No newline at end of file
+msgstr "Click to close the path"
```
Comment (hirunatan) [inline frontend/translations/en.po]: Should add spanish translations, too

**Pull request merged**